config/routes.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.routes.draw do
-   get 'personal_page/index'
-   get 'fun_page/index'
-   get 'social_projects_page/index'
    get 'short_movies/show'
-   root 'build_page#index'
+   get 'pages/fun'
```

Comment:
You should look into resources as wel, you might be able to do 
```
resources :pages do 
  get 'personal`
  ...
```
Try it out, not sure if there are some other quirks to it